vector-stores/models.tsp

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
Not sure if this is worth it, but this is the only API that is organized in a way that matches the documentation. More specifically, their API reference splits "Vector Stores", "Vector Store Files", and "Vector Store File Batches" into separate categories.

Reply:
This feels a bit like Assistants to me where splitting Runs, Threads, Messages, etc. into separate clients seems like a net negative in value. Even if the area's divvied up that way, do you see value in having distinct separation like that?

Reply:
I think you're right. Makes sense to keep them like this for now.